sql/src/main/java/io/crate/execution/dsl/phases/FileUriCollectPhase.java

```diff
@@ -118,6 +128,8 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeString(node);
         }
         Symbols.toStream(toCollect, out);
+        out.writeInt(inputFormat.ordinal());
```

Comment:
This would have to replaced with `out.writeVInt(..)` if you make the change above.